aphrodite/common/sampling_params.py

```diff
@@ -195,6 +211,13 @@ def __init__(
         self.spaces_between_special_tokens = spaces_between_special_tokens
         self.logits_processors = logits_processors or []
         self.include_stop_str_in_output = include_stop_str_in_output
+        if not self.sampler_order:
+            self.sampler_order = ["pens", "temp", "miro", "typ", "quad", "tfs", "minp", "eta", "topa", "topp", "eps", "topk"]
+        self.sampler_order = [[s] if (isinstance(s, str) or isinstance(s, int)) else s for s in self.sampler_order]
+        self.sampler_order = [[_sampler_map[s] if isinstance(s, int) else s for s in sub] for sub in self.sampler_order]
+        self.verify()
```

Comment:
Can we merge the isinstances here?

```py
self.sampler_order = [[s] if isinstance(s, (str, int)) else s for s in self.sampler_order]
self.sampler_order = [[
_sampler_map[s] if isinstance(s, int) else s for s in sub
] for sub in self.sampler_order]
```

Reply:
TIL isinstance can take a tuple of types. Sure, that makes it cleaner, I'll make that change soon.